samples/turtlebot3_bonsai/README.md

```diff
@@ -2,28 +2,36 @@
 This sample provides a Dockerfile and associated code to train reinforecment learning policies for Turtlebot3 using Microsoft Bonsai. 
 
 # Getting Started
-This sample assumes that you have access to an Azure Subscription and have set up a Bonsai workspace. See [Microsoft Bonsai Account Setup](https://docs.microsoft.com/en-us/bonsai/guides/account-setup).
+## Requirements
+* An Azure Subscription with owner permission.
+* A Bonsai workspace. See [Microsoft Bonsai Account Setup](https://docs.microsoft.com/en-us/bonsai/guides/account-setup).
+* [Docker Desktop](https://www.docker.com/products/docker-desktop/).
```

Comment:
Docker Desktop is only required for Windows - do you mind if I make a change in this branch? 

I think specifying docker is important, but installing it is a little different depending on platform.

Reply:
ah! I see. Of course, feel free to push to this branch :)